From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: celenity <celenity@celenity.dev>
Date: Sun, 05 Jul 2026 20:19:20 +0000
Subject: [PATCH] Gecko - Prevent fingerprinting via about:crashes resources

Prevents the browser from exposing that crash reporting is disabled to websites.

Signed-off-by: celenity <celenity@celenity.dev>
---
diff --git a/toolkit/crashreporter/jar.mn b/toolkit/crashreporter/jar.mn
index 641fcaa2fed38..c85a47ec5ca55 100644
--- a/toolkit/crashreporter/jar.mn
+++ b/toolkit/crashreporter/jar.mn
@@ -3,7 +3,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 toolkit.jar:
-#ifdef MOZ_CRASHREPORTER
+#ifndef DER_NOOP
   content/global/crashes.html            (content/crashes.html)
   content/global/crashes.js               (content/crashes.js)
   content/global/crashes.css              (content/crashes.css)
-- 
